src/integration.test.ts

```diff
@@ -43,3 +43,15 @@ describe("siasky.dev end to end integration test", () => {
     );
   });
 });
+
+describe("Registry end to end integration tests", () => {
+  it("Should timeout after the given timeout time in ms", async () => {
+    const { publicKey } = genKeyPairAndSeed();
+
+    // Try getting an inexistant entry.
+    const { entry, signature } = await client.registry.getEntry(publicKey, "foo");
+
+    expect(entry).toBeNull();
+    expect(signature).toBeNull();
+  });
```

Comment:
you don't pass the timeout anywhere

Reply:
I wasn't able to test the timeout really, even for a nonexistant entry the portal got back to me quite fast. I wanted to keep that test but forgot to update the description

I added a test that uses a timeout that is too large and gets rejected, this proves that it gets passed to Sia.